articles/portal/ptl-ref-release-notes.md

```diff
@@ -19,6 +19,17 @@ toc_mdlink: ptl-ref-release-notes.md
 
 This article provides a list of completed Portal releases.
 
+## Release 2.9.0
+
+Date: 1 December 2020
```

Comment:
I'd prefer 'Date: 1st December 2020' which I think is more in keeping with what has gone before

Reply:
Fixed at de1beee0